integration_tests/models/cross_db_utils/test_ceil.sql

```diff
@@ -0,0 +1,12 @@
+
+with data as (
+
+    select * from {{ ref('data_ceil') }}
+
+)
+
+select
+    {{ dbt_utils.ceil(input) }} as actual,
```

Comment:
This needs to be passed as a string — otherwise Jinja is looking for a variable named `input`
```suggestion
    {{ dbt_utils.ceil('input') }} as actual,
```